src/wallet/rpcdump.cpp

```diff
@@ -547,6 +547,7 @@ UniValue dumpwallet(const UniValue& params, bool fHelp)
     if (fHelp || params.size() != 1)
         throw runtime_error(
             "dumpwallet \"filename\"\n"
+            "\nDEPRECATED\n"
```

Comment:
Please add a suggestion to use `z_dumpwallet` instead.

Reply:
`z_exportwallet` is the RPC name